lib/mix/tasks/ecto.drop.ex

```diff
@@ -28,6 +28,18 @@ defmodule Mix.Tasks.Ecto.Drop do
 
     {opts, _, _} = OptionParser.parse args, switches: [quiet: :boolean]
 
+    if disable_safety_warnings?(repo) or Mix.shell.yes?("Are you sure you want to drop the database for repo #{inspect repo}?") do
```

Comment:
Now conf like:
  disable_safety_warnings: "false"
or
  disable_safety_warnings: "true"
is same

Please do explicit comparation with atom true
 
if disable_safety_warnings?(repo)==true or Mix.shell.yes?("Ar